react-cors-spa/src/Theme.js

```diff
@@ -1,14 +1,54 @@
 import { createTheme } from '@mui/material';
 
+/*
+COLOR GUIDE
+Dark teal - #133236
+Not as dark teal - #1a4348
+Yellow - #996d1f
+Red - #b35090
+*/
+
 const theme = createTheme({
   palette: {
     mode: 'dark',
+    background: {
+      default: '#133236',
+      paper: '#1a4348',
+    },
+    primary: {
+      main: '#996d1f',
+    },
+    secondary: {
+      main: '##4a3134',
+    },
+    text: {
+      primary: '#ffffff',
+      secondary: '#ffffff',
+    },
   },
   components: {
+    MuiButton: {
+      styleOverrides: {
+        root: {
+          color: '#ffffff',
```

Comment:
Instead of coloring all buttons this way, you do `<Button color='primary' ...>` to set the color of the button equal to the primary color we defined in L#19. You can do this for any of the named palettes.

Reply:
Ah I was trying to avoid inline styling if possible, and to have a set color scheme for all buttons. However, do you want me to move forward using the theme primary/secondary colors inline?

Reply:
Yeah probably. Or I wonder if you can change the default color here in the theme to primary? That might work and that would be acceptable.